src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -138,6 +130,22 @@ AutomatedRssDownloader::~AutomatedRssDownloader()
     delete m_episodeRegex;
 }
 
+void AutomatedRssDownloader::connectRuleFeedSlots()
+{
+    qDebug() << Q_FUNC_INFO << "Connecting rule and feed slots";
+    connect(ui->listRules, SIGNAL(itemSelectionChanged()), this, SLOT(updateRuleDefinitionBox()), Qt::UniqueConnection);
+    connect(ui->listRules, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleRuleCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+    connect(ui->listFeeds, SIGNAL(itemChanged(QListWidgetItem *)), this, SLOT(handleFeedCheckStateChange(QListWidgetItem *)), Qt::UniqueConnection);
+}
```

Comment:
The `this` pointer is needed in the disconnect (at least it wouldn't compile for me), so I included it in the connect for symmetry.

According to https://doc.qt.io/qt-4.8/qt.html#ConnectionType-enum there shouldn't be a need to also have `Qt::AutoConnection`, but maybe I misunderstood. Could you verify for me please?